torchbenchmark/util/model.py

```diff
@@ -151,7 +151,8 @@ def __post__init__(self):
                     self.set_optimizer(current_optimizer)
         # apply decoration args
         apply_decoration_args(self, self.dargs)
-        eager_latency = warmup(self)
+        if should_check_correctness:
```

Comment:
How does should_check_correctness become different for eager vs pt2? This seems to suggest that we do not check correctness for pt2?

Reply:
We do not check correctness for eager, because we use eager as the baseline to calculate the correctness of pt2.

Reply:
Oh sorry yes I got that flipped. Would it be good to include a comment about that somewhere?

Reply:
Yep I can update when we have the CI result